src/AzSK.ADO/Framework/BugLog/BugLogHelper.ps1

```diff
@@ -263,25 +262,29 @@ class BugLogHelper {
                         if($row.code -eq 200 )
                         {
                             $id = ($row.body | ConvertFrom-Json).id
-                            $tableData = $azTableBugInfo | Where {$_.ADOBugId -eq $id} | Select PartitionKey, projectName
+                            $tableData = $azTableBugInfo | Where {$_.ADOBugId -eq $id} | Select PartitionKey, projectName, ADOScannerHashId
                             #$isDeleted = $this.DeleteTableEntity($tableName, $tableData.partitionKey , $id);
                             #if ($isDeleted -eq $true) {
                             #    $this.AddDataInTable($tableName, $tableData.partitionKey, $id, $tableData.projectName, "Y");
                             #}
+                            $row.body=$row.body.TrimEnd("}")
```

Comment:
try to remove this string operation add desired property in object

Reply:
Cannot do this and write a good common method for both CA and regular SDL Scan closed bugs info collection